portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -517,28 +517,22 @@ definition {
 	@description = "This is a use case for LPS-71236 and LPS-71307. View web content."
 	@priority = "5"
 	test NoXSSVulnerabilityInWebContent {
-		task ("Add CP") {
+		task ("Given: a Web Content with a script in the title is added to the display page") {
```

Comment:
Prefer to break up into two task groups like:
Given: Create a web content with a script in the title.
And: Select web content display in WCD.